install/resources/upgrade41.php

```diff
@@ -35,6 +35,21 @@ function upgrade41_dbchanges()
 	$db->insert_query('spiders', array("name" => "DuckDuckGo", "useragent" => "DuckDuckBot/1.1; (+http://duckduckgo.com/duckduckbot.html)"));
 	$db->insert_query('spiders', array("name" => "UptimeRobot", "useragent" => "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"));
 
-  	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
+	// Remove backslashes from last 1,000 log files.
+	$query = $db->simple_select('moderatorlog', '*', "action LIKE '%\\\\\\%", "limit" => 1000);
+
+	while($row = $db->fetch_array($query))
+	{
+    $original = $row['action'];
+    $stripped = stripslashes($original);
+
+    if($stripped !== $original)
+    {
+        $db->update_query("moderatorlog", "action" => stripped, "WHERE tid = '".$row['tid']."'");
```

Comment:
Missing `$` before `$stripped`. Should maybe cast `$row['tid']` to `(int)` just to be sure.

`$stripped` should also have `$db->escape_string()` ran on it.

Reply:
There's no way for a user to modify 'tid' so it'd just add extra processing time. If you a certain you want it (int)ed I'll make the change, however good catch on $stripped.